src/tab-container/tests/TabContainer.spec.tsx

```diff
@@ -61,8 +61,8 @@ const reverseOrientationTemplate = assertionTemplate(() => (
 
 const tabButtonProperties = {
 	'aria-controls': 'test-tab-0',
-	'aria-disabled': 'false',
-	'aria-selected': 'true',
+	'aria-disabled': 'false' as 'false',
```

Comment:
is this really necessary? Why does `false` need to be cast to `false`?

Reply:
Unfortunately, since TS doesn't infer the correct type here. I've changed it to use the exported `AriaAttributes` interface explicitly though.